ecosystem/sep-0006.md

```diff
@@ -1123,6 +1180,9 @@ Name | Type | Description
 `refunds` | object | (optional) An object describing any on or off-chain refund associated with this transaction. The schema for this object is defined in the [Refunds Object Schema](#refunds-object-schema) section below.
 `required_info_message` | string | (optional) A human-readable message indicating any errors that require updated information from the user.
 `required_info_updates` | object | (optional) A set of fields that require update from the user described in the same format as [/info](#info).  This field is only relevant when `status` is `pending_transaction_info_update`.
+`required_customer_info_message` | string | (optional) A human-readable message indicating any errors that require updated information from the user.
+`required_customer_info_updates` | string | (optional) A set of SEP-9 fields that require update from the user via SEP-12.  This field is only relevant when `status` is `pending_customer_info_update`.
+`instructions` | string | (optional) JSON object containing the [SEP-9 financial account fields](sep-0009.md#financial-account-fields) that describe how to complete the off-chain deposit in the same format as the [/deposit](#deposit) response. This field is only required if `instructions` was not included in the response of a `deposit` request. Only relevant for deposit transactions.
```

Comment:
I think we should change the description to say that `instructions` should be present on the object whether or not it was provided synchronously in the transaction initiation request.

Also, we should state that `instructions` should only be present once the status of the transaction is `pending_user_transfer_start` (not `incomplete` or `pending_cusotmer_info_update`, which are statuses that may proceed it).